eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -261,6 +255,9 @@ function Operate-OnPackages ($activePkgList, $serviceName, $language, $repoRoot,
 
 function Get-PkgListFromYml ($ciYmlPath)
 {
+    $ProgressPreference = "SilentlyContinue"
```

Comment:
I'm pre-approving moving this into the functions but I have a question: Is $ProgressPreference supposed be used somewhere?

Reply:
It was added to prevent the progress bar from displaying while installing the powershell-yaml